docroot/themes/custom/mass_theme/overrides/css/org-page--feedback.css

```diff
@@ -44,6 +67,22 @@ div.ma_feedback-fieldset {
   height: 8px;
 }
 
+@media (max-width: 610px) {
+  .feedback-steps .ma__input-group__items--inline {
+    padding-top: 1px;
+  }
+
+  .feedback-steps .ma__input-group__items--inline .ma__input-radio__label {
+    /* padding-top: 3px; */
```

Comment:
@ygannett can you remove the comment? Since we don't need it.

Reply:
Not anymore.  It's removed.  Thanks.